src/math.rs

```diff
@@ -1,31 +1,45 @@
+#[cfg(not(any(feature = "std", feature = "libm")))]
+compile_error!("either the `std` or `libm` feature is required");
```

Comment:
Is it a general recommendation to put this on top of every module? Otherwise maybe the one on top of lib.rs is enough?

Reply:
Oh, I think I added this to `math.rs` hoping that it would cause the compiler to fatally exit, but ended up having to add hacks anyways to keep the cascading errors out.

I think we can remove this one, good catch!

Reply:
In [e76e265](https://github.com/nathan-osman/rust-sunrise/pull/18/commits/e76e2655fd44c36828ddb0a2cd66d81dcb02d1c4) this is now just at the top of lib.rs.

I believe that that change also fixed all the rest of your review comments (ChangeLog, version bump), and should fix the testsuite.